jekyll/_cci2/how-to-use-the-circleci-local-cli.adoc

```diff
@@ -255,6 +255,20 @@ circleci config process .circleci/config.yml > process.yml
 circleci local execute -c process.yml JOB_NAME
 ```
 
+Note that JOB_NAME is the value of the key-value pair: for example, given 
+
+```yaml
+jobs:
+  test:
+    name: "Test Application"
+```
+
+the invocation would be
+
+```shell
+circleci local execute -c process.yml "Test Application"
+```
```

Comment:
```suggestion
[NOTE]
====
`JOB_NAME` is the value of the key-value pair. For example, given:

[,yaml]
----
jobs:
  test:
    name: "Test Application"
----

The invocation would be:

[,shell]
----
circleci local execute -c process.yml "Test Application"
----
====
```

Reply:
Thank you so much for this @blech! Just a suggestion to make this into a note [admonition](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/). Thank you!